examples/charts/boxplot.py

```diff
@@ -15,9 +15,9 @@
 
 # then, we get the countries and we group the data by medal type
 countries = df.abbr.values.tolist()
-gold = df['medals.gold'].astype(float).values
-silver = df['medals.silver'].astype(float).values
-bronze = df['medals.bronze'].astype(float).values
+gold = list(df['medals.gold'].astype(float).values)
+silver = list(df['medals.silver'].astype(float).values)
+bronze = list(df['medals.bronze'].astype(float).values)
```

Comment:
It was the modified example I used to test.. When I diff before commit I thought "why not"? :disappointed: 
Actually it's a bad idea. Complicates a yet already to nested e data input example. Will revert.

> Does df['medals.silver'] not work?

No. It does not work with pd.Series .. probably something to add as a valid input for the DataAdapter..